.github/workflows/ci.yml

```diff
@@ -36,8 +36,6 @@ jobs:
         exclude:
           - ruby: 3.0
             gemfile: gemfiles/rails_5_2.gemfile
```

Comment:
We can also drop Rails 5.0 and 5.2

Reply:
Right. Will commit it here since you mention it 👍

Reply:
@nbulaj How does it look to you now? 🙂